proto/attestation/reference_value.proto

```diff
@@ -70,9 +70,13 @@ message StringReferenceValue {
 }
 
 message RootLayerReferenceValues {
-  // Switches between AMD and Intel based on TeePlatform value.
-  AmdSevReferenceValues amd_sev = 1;
-  IntelTdxReferenceValues intel_tdx = 2;
+  oneof type {
+    // Switches between AMD SEV-SNP and Intel TDX based on TeePlatform value.
+    // Verification is skipped when not running in a TEE.
+    AmdSevReferenceValues amd_sev = 1;
+    IntelTdxReferenceValues intel_tdx = 2;
```

Comment:
Would the client know/care whether running on AMD or Intel hardware? Consequently the client would need to pass reference values for all platforms at once. Put differently, the lack of oneof was by intention.

Reply:
Good point. Reverted and added `InsecureReferenceValues` to support non TEE platforms.